packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -58,70 +59,160 @@ export interface TcpTimeout {
 }
 
 /**
- * Enum of supported AppMesh protocols
+ * Properties used to define healthchecks.
  */
-export enum Protocol {
-  HTTP = 'http',
-  TCP = 'tcp',
-  HTTP2 = 'http2',
-  GRPC = 'grpc',
-}
-
-/**
- * Properties used to define healthchecks when creating virtual nodes.
- * All values have a default if only specified as {} when creating.
- * If property not set, then no healthchecks will be defined.
- */
-export interface HealthCheck {
+export interface HealthCheckCommonOptions {
   /**
-   * Number of successful attempts before considering the node UP
+   * The number of consecutive successful health checks that must occur before declaring listener healthy.
    *
-   * @default 2
+   * @default 5
    */
   readonly healthyThreshold?: number;
 
   /**
-   * Interval in milliseconds to re-check
+   * The time period in milliseconds between each health check execution.
```

Comment:
Let's get rid of the "milliseconds" here while we're changing these (the type here is `Duration`, no need to burden the customer with the details of what unit is required).

Reply:
Good catch. Will do